README.md

```diff
@@ -1,22 +1,51 @@
-# Instrucciones para crear Favicons
+# Bases de buenas prácticas en sitios web EnFlujo 💅🏽
+----
 
-Instrucciones para crear favicons para los sitios web. Basado en este excelente [artículo](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs): 
+## 1. Fuentes suplentes o _font fallbacks_
 
-Estos son los elementos que van en el `<head>` del html:
+En el espcacio en blanco (__________), va el nombre de la tipografía escogida, que será nuestra primera opción. Cuando esa fuente se demore o no se pueda cargar por alguna razón, el navegador debería usar las que están a la derecha.
+
+El nombre `--fuente` corresponde al nombre de variable que le estamos dando a cada una de las fuentes.
+
+Para cada tipo de fuente se debería poner algo así (el nombre de la variable puede cambiar):
+
+* Monoespaciadas
+
+
+`--fuenteMonoespaciada: '______________', 'Courier New', monospace;`
+
+* Serif
+
+`--fuenteSerifada: '______________', 'Garamond', 'Georgia', 'Times', serif;`
+
+* Sans-serif
+
+`--fuenteSinSerifas: '______________',  'Helvetica', 'Roboto', 'Arial', 'Tahoma', 'Verdana', sans-serif;`
+
+
+----
+
+
+## 2. Instrucciones para crear Favicons
+
+Basado en este excelente [artículo](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs): 
+
+Estos son los elementos que van en el `<head>` del HTML:
 
 ```html
-<link rel="icon" href="/favicon.ico" sizes="any"><!-- 500x500 -->
-<link rel="icon" href="/favicon.svg" type="image/svg+xml"><!-- 36x36 -->
-<link rel="apple-touch-icon" href="/apple-touch-icon.png"><!-- 180×180 -->
+<link rel="icon" href="/favicon.ico" sizes="32x32">
```

Comment:
este debe ir con `"any"` como estaba antes.

Reply:
Pero pille que como que lo actualizaron a "32x32" en el ChangeLog del artículo:

**Changelog**
2023-07-11
sizes="any" for ICO file was replaced to sizes="32x32" to fix bug when Chrome downloaded both ICO and SVG files.

https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs#:~:text=Changelog,and%20SVG%20files.